portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -63,6 +63,28 @@ definition {
 			entryType = "${customElementType}");
 	}
 
+	@description = "Verify that remote app of type Custom Element can be deleted"
+	@priority = "4"
+	test CustomElementCanBeDeleted {
+		property portal.acceptance = "true";
+
+		var customElementName = "Vanilla Counter";
+		var customElementType = "Custom Element";
+		var customElementHTMLName = "vanilla-counter";
+		var customElementURL = "http://remote-component-test.wincent.com/packages/vanilla-counter/index.js";
+
+		RemoteApps.goToRemoteAppsPortlet();
+
```

Comment:
Need to add `task ( )` to define the main steps of the test.

Also, please use JSON macros for Setup phases of the test. ie: Anything before the main action/assertion of test 
(delete the remote app, and assert not present) are setup (add remote app). So need to use JSON as much as possible in setup phase to make test more efficient with speed and avoiding unrelated test failures with setup (ie: UI for adding a remote app).

Please reach out to @magjed4289 if need help with JSON, but there should already be some examples of this in other Remote App tests.

cc/ @Esther-OC